packages/node-api/src/tx.ts

```diff
@@ -80,21 +80,21 @@ function extractEvents(api: ApiPromise, result: ISubmittableResult): IExtractEve
 
                             message = `${error.section}.${error.name}`;
                         } catch (error) {
-                            message = `could not extract dispatch error: ${error}`;
+                            message = `could not extract dispatch error: ${JSON.stringify(error)}`;
                         }
                     }
-                    console.log('txUpdateEvent', {
+                    console.log('txUpdateEvent', `${JSON.stringify({
                         title: `${section}.${method}`,
                         message
-                    });
+                    })}`);
                     error = message;
                 }
 
             } else {
-                console.log('txUpdateEvent', {
+                console.log('txUpdateEvent', `${JSON.stringify({
```

Comment:
This worked in the tests here, but in the wallet-app, only `[object]` was printed.